library/lib/REST.php

```diff
@@ -69,17 +69,17 @@ class REST
     protected $mime_type;
 
 
-    function __construct($config = array())
+    function __construct($config = array(), $http_options = array())
```

Comment:
Based on making things work this is fine and seems to fall into their pattern.

Ideally, we are injecting `ClientInterface` here which would allow someone to pass in their preconfigured HTTP client.